Makefile

```diff
@@ -1,5 +1,6 @@
-FAKTORY_HOST=127.0.0.1
+FAKTORY_HOST=localhost
```

Comment:
Should be `localhost` since the ssl certificate is for this common name. Does not affect the tests or local development in any way.